### DISCUSSION TITLE
PENDING REDEEM

### DISCUSSION THREAD

**Discussion by nghiaeuro123** [wormhole-foundation/wormhole#3804] [Developer Q&A] [answered]
I bridged 14.99APT from BEP20 network to APT on https://portalbridge.com/at 4:43 p.m. on January 20, 2024. But until now I still have not received the aptos network APT token (March 1, 2024)
When I looked it up again, it said PENDING REDEEM on the website
Tx Hash: [0x90492e5dd973d1074a02075ded0c64af73b061a0be401c7ef1e4782306628678](https://bscscan.com/tx/0x90492e5dd973d1074a02075ded0c64af73b061a0be401c7ef1e4782306628678)
<img width="1440" alt="2" src="https://github.com/wormhole-foundation/wormhole/assets/153918734/57b87d56-364c-4d84-af63-f632160089d8">

**Comment by aadam-10** [comment, marked as answer]
hello, please use discord for faster assistance: https://discord.gg/wormholecrypto
